pool/src/auctions/bad_debt_auction.rs

```diff
@@ -47,11 +48,10 @@ pub fn create_bad_debt_auction_data(e: &Env, backstop: &Address) -> AuctionData
     // TODO: Remove backstop_token getter call if WASM cache is not implemented
```

Comment:
can we drop this TODO? this fits under limits now so it is not a large concern.

Reply:
We talked offline about this and we will ensure all TODOs are managed in a separate PR.